src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -423,6 +441,28 @@ export default class BrowserCell extends Component {
       classes.push(styles.required);
     }
 
+    let extras = null;
+    if (this.state.showConfirmationDialog)
+      extras = (
+        <Modal
+          type={this.selectedScript.type === 'info' ? Modal.Types.INFO : Modal.Types.DANGER}
```

Comment:
What's the param name for the confirmation dialog type in the config file? If it's just `type` could you change it to confirmationDialogStyle`?

Reply:
nice finding. It is wrong. I forgot to change it when refactored. Thanks!